packages/rum-core/src/domain/rumEventsCollection/resource/resourceCollection.ts

```diff
@@ -130,12 +137,25 @@ function computeRequestTracingInfo(request: RequestCompleteEvent) {
     _dd: {
       span_id: request.spanId!.toDecimalString(),
       trace_id: request.traceId!.toDecimalString(),
+      rule_psr: configuration.tracingSampleRate / 100, // rule_psr needs to be between 0-1
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏add a function like `function getRulePsr(configuration)` to keep the computation only once